src/askscript/__tests__/13-real-operators/real-operators-01e-exclamation.ask

```diff
@@ -0,0 +1,3 @@
+ask { // Implemented in https://github.com/xFAANG/askql/issues/295
```

Comment:
Please add more test cases, eg.

`a != 0 && a != 1`

And add expected test result

Reply:
This would be a good test for https://github.com/xFAANG/askql/issues/296 (Add != operator) but the current ticket was only about adding '!' to the operators in the grammar, so the only good test is to check that .ask file parses successfully (and it does).